002-routing-params-views/src/app/states/auth/auth.route.js

```diff
@@ -12,20 +12,29 @@
         abstract: true,
         template: '<section class="general-section" ui-view></section>',
         url: '/auth',
+        data: {
+          requiresLogin: false
+        }
       })
       .state('login', {
       	parent: 'auth',
       	url: '/login',
         params: {
           currentUser: {}
         },
+        data: {
+          requiresLogin: false
+        },
         templateUrl: 'app/states/auth/login/login.html',
         controller: 'AuthLoginController',
         controllerAs: 'auLoginCtrl'
       })
        .state('signin', {
       	parent: 'auth',
       	url: '/signin',
+        data: {
+          requiresLogin: false
+        },
         templateUrl: 'app/states/auth/signin/signin.html',
         controller: 'AuthSigninController',
         controllerAs: 'auSigninCtrl'
```

Comment:
this is going to become a huge ball of mud (imagine 30 views configurations, with complex resolve each in one single file). Split this route in to several ideally one for each view, so inside each view/state folder there will be at least 3 files: <name>.html, <name>.controller.js, <name>.route.js. So not only main states all of them have they're own routes file